v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -209,5 +213,47 @@ def requires(self) -> ReferenceDataset:
             RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/4.0/pca/gnomad.v4.0.pca_loadings.ht',
         },
     },
+    ReferenceDataset.gnomad_mito: {
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.MITO]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/3.1/ht/genomes/gnomad.genomes.v3.1.sites.chrM.ht',
+        },
+    },
+    ReferenceDataset.gnomad_non_coding_constraint: {
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://seqr-reference-data/GRCh38/gnomad_nc_constraint/gnomad_non-coding_constraint_z_scores.ht',
+        },
+    },
+    ReferenceDataset.screen: {
+        ENUMS: {
+            'region_type': [
+                'CTCF-bound',
+                'CTCF-only',
+                'DNase-H3K4me3',
+                'PLS',
+                'dELS',
+                'pELS',
+                'DNase-only',
+                'low-DNase',
+            ],
+        },
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://seqr-reference-data/GRCh38/ccREs/GRCh38-ccREs.ht',
```

Comment:
Same as above

Reply:
ah there's a script that I missed to create the ht from the .bed file, which I need to incorporate into get_ht  for `screen`

Reply:
screen is updated!